packages/design-system-tailwind-preset/package.json

```diff
@@ -51,13 +51,14 @@
     "@types/jest": "^27.4.1",
     "deepmerge": "^4.2.2",
     "jest": "^27.5.1",
-    "tailwindcss": "^3.4.13",
```

Comment:
Moving to `peerDependencies`

Reply:
oh interesting - i thought you needed to do have both because dev will use version you have define here and peerDependency would use the defined version. but I looked it up and understand the duplication situation of potentially having two installs when working locally